sources/packages/backend/workflow/test/2022-2023/parttime-assessment/appeals/parttime-assessment-student-financial-information.e2e-spec.ts

```diff
@@ -192,6 +192,36 @@ describe(`E2E Test Workflow parttime-assessment-${PROGRAM_YEAR}-student-financia
     },
   );
 
+  it(
+    "Should have calculated current year partner income in student financial information " +
+      "when there is a request a change for current year partner income.",
+    async () => {
+      // Arrange
+      const assessmentConsolidatedData =
```

Comment:
In the consolidated data, please make sure `partner1CRAReportedIncome` is populated. So that we can check the output for the precedence of `calculatedDataCurrentYearPartnerIncome`

`calculatedDataPartner1TotalIncome`

Reply:
Also I would recommend to have one more test at-least to validate the precedence of  `calculatedDataCurrentYearPartnerIncome` over `calculatedDataPartner1TotalIncome`